remotes/fixup.go

```diff
@@ -136,3 +179,55 @@ func checkBaseImage(baseImage *bundle.BaseImage) error {
 
 	return nil
 }
+
+// FixupEvent is an event that is raised by the Fixup Logic
+type FixupEvent struct {
+	SourceImage    string
+	DestinationRef reference.Named
+	Descriptor     *ocischemav1.Descriptor
+	EventType      FixupEventType
+	Message        string
+	Error          error
+	DigestTree     *DigestTree
```

Comment:
Are all fields (apart from `Error` which I see documented) always populated?

(nit: I'd put the event type first in the struct...)

Reply:
DigestTree is for now populated only on DigestTreeResolved event. I can change that.